Update Sprig list (#7843)
open-cluster-management/rhacm-docs: clusters/about/cluster_mce_overview.adoc

The {mce-short} is the cluster lifecycle operator that provides cluster management capabilities for {ocp} and {acm-short} hub clusters. If you installed {acm-short}, you do not need to install {mce-short}, as it is automatically installed. See other important information about Cluster lifecycle with {mce-short}:

> - Your cluster is created by using the {ocp-short} cluster installer with the Hive resource. See more information about the process of installing {ocp-short} clusters at link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/installation_overview/index[Installing and configuring {ocp-short} clusters] in the {ocp-short} documentation.   

Reviewer: Is it possible not to "hard-coded" / reference a specific OCP version? 4.16 is kind of old. 😅

Reviewer: If choose to update, there are a few more places that reference to `4.16` below.

Author: Yes, but when I raised it in the past having a reference wasn't compatible with the current doc processes. Not sure whether that stance has changed.

Reviewer 2: I update these links each release, but the reason we cannot rely on a hard code is because some of the links have to be a specific version. We mostly just find and replace, I just had not gotten to it yet.

> - Your cluster is created by using the {ocp-short} cluster installer with the Hive resource. You can find more information about the process of installing {ocp-short} clusters at link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/installation_overview/index[Installing and configuring {ocp-short} clusters] in the {ocp-short} documentation.   

Reviewer 3: Simple recommended change to be consistent with how we typically introduce links: 

```suggestion
- Your cluster is created by using the {ocp-short} cluster installer with the Hive resource. See more information about the process of installing {ocp-short} clusters at link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/installation_overview/index[Installing and configuring {ocp-short} clusters] in the {ocp-short} documentation.   
```
